G/Gnuplot/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+
+using BinaryBuilder, Pkg
+
+name = "gnuplot"
+version = v"5.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://excellmedia.dl.sourceforge.net/project/gnuplot/gnuplot/5.4.1/gnuplot-5.4.1.tar.gz", "6b690485567eaeb938c26936e5e0681cf70c856d273cc2c45fabf64d8bc6590e"),
```

Comment:
Use $version, move checksum to second line?

Reply:
Okay, done!